pkgs/tools/security/regpg/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchgit
+, git
+, perl
+, perlTextMarkdown
```

Comment:
```suggestion
, TextMarkdown
```

Reply:
or prefer to build a custom perl env with `perl.withPackages`, which probably be even better.